aws/cloudtrail/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
I don't want this script duplicated into each project. If we need that, then move it to `geodesic`